### DISCUSSION TITLE
Why not integrate OnlyR into OnlyT

### DISCUSSION THREAD

**Discussion by Psudonico** [AntonyCorbett/OnlyT#472] [Ideas]
Hello Brother, 
what do you think about integrating OnlyR into OnlyT?
Maybe making the registrations automated at the start of the gathering.
Thanks

**Comment by AntonyCorbett** [comment]
@Psudonico Thanks for your suggestion. I deliberately separated these functions which did at one time exist together in SoundBox software. I think this way offers more flexibility.
